backend/app/services/chat/trigger/core.py

```
@@ -406,6 +406,26 @@ async def _stream_chat_response(
+            if memory_service.enabled:
+                memory_context = await memory_service.retrieve_for_chat(
+                    user_id=stream_data.user_id,
+                    team_id=stream_data.team_id,
+                    task_id=stream_data.task_id,
+                    query=message,
+                )
+                if memory_context:
+                    enhanced_system_prompt = (
+                        f"{enhanced_system_prompt}\n\n{memory_context}"
+                    )
+                    logger.info(f"[ai_trigger] Injected long-term memory context")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)